cstore_fdw.c

```diff
@@ -299,8 +307,9 @@ CStoreProcessUtility(Node * parseTree, const char *queryString,
 
 		if (dropStmt->removeType == OBJECT_EXTENSION)
```

Comment:
`DROP EXTENSION` should also fire our `cstore_drop_trigger()` in cstore_fdw--1.5.sql. Why not implement all these in there? (See the matrix at https://www.postgresql.org/docs/current/static/event-trigger-matrix.html)